opts/hosts_test.go

```diff
@@ -53,8 +53,8 @@ func TestParseHost(t *testing.T) {
 func TestParseDockerDaemonHost(t *testing.T) {
 	invalids := map[string]string{
 
-		"tcp:a.b.c.d":                   "Invalid bind address format: tcp:a.b.c.d",
-		"tcp:a.b.c.d/path":              "Invalid bind address format: tcp:a.b.c.d/path",
+		"tcp:a.b.c.d":                   "",
+		"tcp:a.b.c.d/path":              "",
```

Comment:
For future reference; I decided to keep the change minimal here. The error here is now coming from Golang, so I'm not checking on the actual error string (in case it changes); it's not really important, as long as it's marked "invalid".

There's some cleaning-up to do in these functions (and I can improve things as part of that)